### PR TITLE
ENH: Finance Case Study blog post

### PR DESCRIPTION
**Pull request by Ahtritus**
## What

Please include a summary of the changes.

- [x] Added item
- [ ] Updated item

## How

Added new blog post

## Why

New article

## Screenshots (If applicable)

Please include any relevant screenshots that may help explain the change.

## Notes

Notes about the change, if any.

# Checklist

- [x] My code follows the style guidelines of this project and is properly formatted
- [x] I have performed a self-review of my own changes
- [x] All existing tests pass, and I have added new ones if required

### CODE REVIEW

**Comment by Ahtritus** [comment]
Not sure why build is failing, perhaps because of multiple authors. @2KAbhishek

**Comment by 2KAbhishek** [comment]
Multiple author is broken after the recent hugo upgrade, should just limit it to one for now.
Or you can dig in and add a fix :)

File: content/blog/cats-case-study.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+### Client:
+
+A leading customer credit provider in the United States, committed to assisting customers in achieving their personal objectives through great customer service and flexible lending alternatives.
+
+### Problem statement:
+
+The bank has over 30 million unique loan contracts spread across various departments. As the number of contracts grew over time, handling all these contracts manually led to these major drawbacks.
+
+1. No centralized system to access all the contracts and work on them. The agents had to do multiple logins to different branch systems to work on these contracts and search across various branches even find the correct contract.
+2. While working on a contract and doing multiple operations, manual errors inevitably started creeping in.
```
Comment (akshayvadher) [inline content/blog/cats-case-study.md:22]: How to say it better - the word "multiple operations"? It is multiple actions **for** **one event**.  From reader's perspective multiple operations mean doing multiple actions but for us, it is multiple "button clicks" for doing "one operations"

File: content/blog/cats-case-study.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+<h2>~2X</h2> &nbsp;&nbsp; increase in contracts handled daily
+</span>
+
+<span style="display: flex;flex-direction: row;align-content: center;justify-content: flex-start;align-items: center;">
+<h2>~160x</h2> &nbsp;&nbsp; daily users across 5 different departments
+</span>
+
+### Challenges:
+
+1. **The client had been using an outdated CRM which was hosted on on-prem servers.** This involved huge maintenance costs for servers and a separate IT department. As the number of agents and contracts kept on increasing, cloud seemed to be the correct choice going forward.
```
Comment (akshayvadher) [inline content/blog/cats-case-study.md:51]: Is outdated proper word? Is there anything non offensive word? Like legacy?

File: content/blog/cats-case-study.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+### Client:
+
+A leading customer credit provider in the United States, committed to assisting customers in achieving their personal objectives through great customer service and flexible lending alternatives.
+
+### Problem statement:
+
+The bank has over 30 million unique loan contracts spread across various departments. As the number of contracts grew over time, handling all these contracts manually led to these major drawbacks.
+
+1. No centralized system to access all the contracts and work on them. The agents had to do multiple logins to different branch systems to work on these contracts and search across various branches even find the correct contract.
```
Comment (akshayvadher) [inline content/blog/cats-case-study.md:21]: Not only different branches but different systems for different workflows as well. And the same in the solution - all workflows in the same system. 
One system to rule them all.

**Comment by akshayvadher** [comment]
> Multiple author is broken after the recent hugo upgrade, should just limit it to one for now. Or you can dig in and add a fix :)

image
Working for me in local. Can you please point me to the issue anywhere?

**Comment by 2KAbhishek** [comment]
Try checking if the build logs have anything

**Comment by akshayvadher** [comment]
> Try checking if the build logs have anything

image
Nothing

**Comment by 2KAbhishek** [comment]
> > Try checking if the build logs have anything
> 
> image Nothing

https://github.com/incubyte/incubyte.github.io/actions/runs/4233393155/jobs/7354302590 I believe this could be because of version mismatch

Also can you try `hugo --minify` on local, that's what we are using on GitHub workflow.

we can potentially look at removing the --minify from github build, potential drawback: slightly longer load times

File: content/blog/cats-case-study.md (hunk 1 above)
Comment (Ahtritus) [reply content/blog/cats-case-study.md]: Changed to include this

File: content/blog/cats-case-study.md (hunk 2 above)
Comment (Ahtritus) [reply content/blog/cats-case-study.md]: Using `legacy`

File: content/blog/cats-case-study.md (hunk 3 above)
Comment (Ahtritus) [reply content/blog/cats-case-study.md]: lol. updated slightly

**Comment by Ahtritus** [comment]
> > > Try checking if the build logs have anything
> > 
> > 
> > image Nothing
> 
> https://github.com/incubyte/incubyte.github.io/actions/runs/4233393155/jobs/7354302590 I believe this could be because of version mismatch
> 
> Also can you try `hugo --minify` on local, that's what we are using on GitHub workflow.
> 
> we can potentially look at removing the --minify from github build, potential drawback: slightly longer load times

Removed the --minify, build succeded. @2KAbhishek you wanna raise a separate PR for this?

**Comment by 2KAbhishek** [comment]
Multiple authors fixed on #119

**Review by 2KAbhishek — CHANGES_REQUESTED** [review]
We should avoid using CATS in the file name, also the slug should not change as  we may have already shared the links with potential clients

**Comment by AkashG6** [comment]
Two corrections needed in CATS case study file,
line 19, over 3.6M*
line 66, latest*

**Review by 2KAbhishek — APPROVED** [review]

**Pull request merged**